client/locations-list-map.js

```diff
@@ -1528,6 +1536,9 @@ function initMapSearch(data, filters) {
     e.preventDefault();
     resetMap(data, filters);
     $search.val('');
+    Object.keys(filterSelectors).forEach((current) => {
+      filterSelectors[current].clear();
```

Comment:
Hmm... so getting to this line, it looks like what we're trying to do is just call .clear() on all selectors. Is that accurate?

If so, do we actually need a map of the selector id to the selectr object? Could we get away with just an array of all selectr objects to clear?

Reply:
Hey Albert, thanks for the review! I kept it a map of filter id to selector objects in case someone deselects a selector. So the map only contains the currently selected ones, I can look for a way to achieve this with an array too.